### DISCUSSION TITLE
*Solved*Game freezes immediately after Running War of The Chosen through AML with LWoTC and its dependencies enabled

### DISCUSSION THREAD

**Discussion by Rosenkov** [X2CommunityCore/xcom2-launcher#359] [Help] [answered]
**AML Version:*1.5.0* 

**Description**
*Clean Reinstall of both AML and XCOM2 for LWoTC, Game freezes immediately after Running War of The Chosen through AML with LWoTC and its dependencies enabled*

 If anyone can help, I'd appreciate it very much.
![image](https://github.com/X2CommunityCore/xcom2-launcher/assets/152428868/e178797a-3435-4da9-bcc7-9b3a18cb5f1a)
This is the state right after I clicked run wotc through AML
XCOM2 is updated to latest
And 1.5.0 is the latest of AML. 
Double checked the mods updates, they're all updated according to AML.

**Comment by Iridar** [comment, marked as answer]
This issue doesn't have anything to do with AML. Follow standard mod troubleshooting steps: https://www.reddit.com/r/xcom2mods/wiki/mod_troubleshooting#wiki_my_game_freezes_on_launch.21

Start by testing if the game will launch without mods, then if it will launch with just the Highlander active - most likely it won't, in that case follow the Highlander troubleshooting, linked on the Highlander's workshop page.

**Comment by Rosenkov** [comment]
> This issue doesn't have anything to do with AML. Follow standard mod troubleshooting steps: https://www.reddit.com/r/xcom2mods/wiki/mod_troubleshooting#wiki_my_game_freezes_on_launch.21
> 
> Start by testing if the game will launch without mods, then if it will launch with just the Highlander active - most likely it won't, in that case follow the Highlander troubleshooting, linked on the Highlander's workshop page.

Oh okay, I'll try that later. Thanks for the help!
